README.md

```diff
@@ -301,8 +301,32 @@ spec:
 
 You can also achieve persistence by creating a PV in a mounted host folder.
 
-## Mounted Host Folders
-Some drivers will mount a host folder within the VM so that you can easily share files between the VM and host.  These are not configurable at the moment and different for the driver and OS you are using.  Note: Host folder sharing is not implemented in the KVM driver yet.
+## Mounting Host Folders
+`minikube mount /path/to/directory` is the recommended way to mount directories into minikube so that they can be used in your local kubernetes cluster.  The command works on all platforms.   Below is an example pod that reads from, creates and removes files from the mounted directory on the host folder `/mount-9p`.
+
+```yaml
```

Comment:
Maybe instead of the YAML, you could show a workflow with the `minikube mount` commands?